test/kb_blast_server_test.py

```diff
@@ -172,28 +399,40 @@ def test_kb_blast_BLASTp_Search_01_Genome(self):
         self.assertIsNotNone(report_obj['objects_created'][0]['ref'])
 
         created_obj_0_info = self.getWsClient().get_object_info_new({'objects':[{'ref':report_obj['objects_created'][0]['ref']}]})[0]
-        [OBJID_I, NAME_I, TYPE_I, SAVE_DATE_I, VERSION_I, SAVED_BY_I, WSID_I, WORKSPACE_I, CHSUM_I, SIZE_I, META_I] = list(range(11))  # object_info tuple
-        self.assertEqual(created_obj_0_info[NAME_I], target_1+'-'+obj_out_name)
+        self.assertEqual(created_obj_0_info[NAME_I], obj_out_name)
         self.assertEqual(created_obj_0_info[TYPE_I].split('-')[0], obj_out_type)
+
+        # check number of hits in featureSet output
+        featureSet_out_obj = self.getWsClient().get_objects([{'ref':report_obj['objects_created'][0]['ref']}])[0]['data']
+        self.assertEqual(expected_hit_cnt, len(featureSet_out_obj['element_ordering']))
         pass
 
 
-    # Test BLASTp: GenomeSet
+    # Test BLASTp: GenomeSet target
     #
     # Uncomment to skip this test
     # HIDE @unittest.skip("skipped test_kb_blast_BLASTp_Search_02_GenomeSet")
     def test_kb_blast_BLASTp_Search_02_GenomeSet(self):
         [OBJID_I, NAME_I, TYPE_I, SAVE_DATE_I, VERSION_I, SAVED_BY_I, WSID_I, WORKSPACE_I, CHSUM_I, SIZE_I, META_I] = range(11)  # object_info tuple
 
-        target_1 = 'test_genomeset'
         obj_basename = 'BLASTp_GenomeSet'
         obj_out_name = obj_basename+".test_output.FS"
         obj_out_type = "KBaseCollections.FeatureSet"
+        genomeSet_name = 'test_genomeSet.BLASTp.GenomeSet'
+        expected_hit_cnt = 2
+
+        #reference_prok_genomes_WS = 'ReferenceDataManager'  # PROD and CI
+        #genome_ref_1 = 'ReferenceDataManager/GCF_001566335.1/1'  # E. coli K-12 MG1655
+        #genome_ref_2 = 'ReferenceDataManager/GCF_002936495.2/1'  # E. coli
+        #genome_ref_3 = 'ReferenceDataManager/GCF_002936145.2/1'  # E. coli
+
+        genomeInfo_0 = self.getGenomeInfo('GCF_001566335.1_ASM156633v1_genomic', 0)  # E. coli K-12 MG1655
+        genomeInfo_1 = self.getGenomeInfo('GCF_000021385.1_ASM2138v1_genomic', 1)    # D. vulgaris str. 'Miyazaki F
+        genomeInfo_2 = self.getGenomeInfo('GCF_001721825.1_ASM172182v1_genomic', 2)  # Pseudomonas aeruginosa
+        genome_ref_1 = self.get_obj_ref_from_obj_info(genomeInfo_0)
+        genome_ref_2 = self.get_obj_ref_from_obj_info(genomeInfo_1)
+        genome_ref_3 = self.get_obj_ref_from_obj_info(genomeInfo_2)
```

Comment:
rm comments and see previous comment about making similarly-named variables.

Reply:
done